docs/index.rst

```diff
@@ -207,50 +209,15 @@ This command will generate a ``PostFactory`` class that looks like this:
                 zenstruck_foundry:
                     make_factory:
                         default_namespace: 'App\\MyFactories'
-                    make_story:
-                        default_namespace: 'App\\MyStories'
```

Comment:
shouldn't we keep this? based on https://github.com/zenstruck/foundry/issues/516#issuecomment-2033692112 I think we should (and upmerge the related PR)